SECURITY.md

```diff
@@ -32,7 +32,9 @@ Support for OpenSSL is only provided for the Netty transport via [netty-tcnative
 
 As of version `1.1.33.Fork14`, netty-tcnative provides two options for usage: statically or dynamically linked. For simplification of initial setup,
 we recommend that users first look at `netty-tcnative-boringssl-static`, which is statically linked against BoringSSL and Apache APR. Using this artifact requires no extra installation and guarantees that ALPN and the ciphers required for
-HTTP/2 are available.
+HTTP/2 are available. In addition, starting with `1.1.33.Fork16` binaries for
```

Comment:
I'd rather say Fork16 since that is when the uber jar began to exist. Conflating the statement by saying Fork17 would be counter-productive if I was reading the documentation.

Instead, we suggest to use Fork17 later in the docs. If we wanted to explicitly state there was a problem with Fork16 on Windows, that could be fine, but I was thinking that may be getting to be a bit much.